src/olympia/accounts/helpers.py

```diff
@@ -90,3 +90,7 @@ def generate_fxa_state():
 def camel_case(snake):
     parts = snake.split('_')
     return parts[0] + ''.join(part.capitalize() for part in parts[1:])
+
+
+def redirect_for_login(request):
+    return HttpResponseRedirect(default_fxa_login_url(request))
```

Comment:
I find the distinction between causes a lot of circular imports. I'll try moving it though.